conjur/list/list_data.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+"""
+ListData module
+
+This module represents the DTO that holds the params the user passes in.
+We use this DTO to build the list request
+"""
+
+# pylint: disable=too-few-public-methods
+class ListData:
+    """
+    Used for organizing the the params the user passed in to execute the list command
+    """
+    # pylint: disable=line-too-long,too-many-arguments
+    def __init__(self, kind=None, inspect=False, search=None, limit=None, offset=None, acting_as=None):
+        self.kind = kind
+        self.inspect = inspect
+        self.search = search
+        self.limit = limit
+        self.offset = offset
+        self.acting_as = acting_as
+
+    def list_dictify(self):
+        """
+        Method for building a dictionary from all attributes that have values
+        """
+        list_dict={}
+        for attr, value in self.__dict__.items():
+            if value is not None and value is not False:
```

Comment:
```suggestion
            if value:
```
but isn't False sometimes a valid input?

Reply:
inspect is a special case where it is either true or false. Because of this it will always be present in the list without this check. This `list_dict` is an accumulation of all the params that the user actively inputted at the command line. Because inspect will always be present even if it is False we don't want to count False as active user input

Reply:
nice check 👌  changing to `if value`